src/index.js

```diff
@@ -0,0 +1,136 @@
+const state = {
+  temperature: 0,
+  city: 'Seattle',
+};
+
+// Initial page load
+const pageInitialLoad = () => {
+  getCityLoc('Seattle');
+  cityNameField.value = 'Seattle';
+};
+
+// helper variables to create event listeners
+const cityNameField = document.getElementById('cityNameField');
+const cityNameDisplay = document.getElementById('cityNameDisplay');
+const temperatureContainer = document.getElementById('temperature');
+const addTempButton = document.getElementById('increaseTemp');
+const minusTempButton = document.getElementById('decreaseTemp');
+const resetButton = document.getElementById('reset');
+const currentTemperatureButton = document.getElementById('currentTemp');
+const skyDropdown = document.getElementById('skyDropdown');
+const emojiContainer = document.getElementById('emojiContainer');
```

Comment:
This works, but only because the tag in your HTML that is including the JS script is at the end of the HTML file.  If this was moved up to the head section of the HTML, which is also a common place to put the script, these would all be `undefined` because the code here would be running before any of these DOM elements are created.  If you want to create global variables like this, I strongly recommend creating a global object similar to the `state` object above, and then initializing all of these variables inside of that object in an event listener like `DOMContentLoaded` on line 94.